arangod/RestHandler/RestAdminClusterHandler.h

```diff
@@ -209,7 +209,8 @@ class RestAdminClusterHandler : public RestVocbaseBaseHandler {
   FutureVoid handlePostRebalanceShards(const ReshardAlgorithm&);
 
   cluster::rebalance::AutoRebalanceProblem collectRebalanceInformation(
-      std::vector<std::string> const& excludedDatabases);
+      std::vector<std::string> const& excludedDatabases,
+      bool excludeSystemCollections = false);
```

Comment:
Let's try to avoid default values, and have all callers pass the explicit value instead.
```suggestion
      bool excludeSystemCollections);
```